src/tracker.js

```diff
@@ -13,8 +13,11 @@ const defaultPost = (url, body) => fetch(url, {
 });
 
 export const init =
-  ({url, application, browser}, post = defaultPost, log = defaultLog) =>
-    async ({type, payload, platform, user:{id}, uri}) => {
+  ({url, application, browser}, post = defaultPost, log = defaultLog) => {
+    if (!url) {
+        return async () => void(0)
```

Comment:
you can `log` something here, indicating that no url was defined (use logger provided as parameter)